modules/security-context-constraints-command-reference.adoc

```diff
@@ -3,20 +3,16 @@
 // * authentication/managing-security-context-constraints.adoc
 
 [id="security-context-constraints-command-reference_{context}"]
-= Security context constraints reference commands
+= Reference of security context constraints commands
 
-You can manage SCCs in your instance as normal API objects using the CLI.
+You can manage security context constraints (SCCs) in your instance as normal API objects using the OpenShift CLI (`oc`).
 
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 [NOTE]
 ====
 You must have `cluster-admin` privileges to manage SCCs.
 ====
 
-[IMPORTANT]
```

Comment:
just asking - was this moved elsewhere or did it appear twice on the page because of the modularization?

Reply:
Yeah, it appeared several places. I would have kept it and moved it below to the updating SCCs section below, but it already has a similar warning there. If you prefer this wording, I can move this one down and replace the other one with it.